plugins/inventory/aws_ec2.py

```diff
@@ -456,7 +478,7 @@ def _get_instances_by_region(self, regions, filters, strict_permissions):
 
             all_instances.extend(instances)
 
-        return sorted(all_instances, key=lambda x: x['InstanceId'])
+        return all_instances
```

Comment:
While Instance ID isn't necessarily the right thing to sort by, dropping the sort may cause problems for anyone who relied on this returning a predictable inventory.  By default Ansible will honour the order that hosts are listed in the inventory.

Reply:
I would like to preserve the order. I just moved the line in `_query()`.

Reply:
Ah, ok I missed that :)